client/components/order-packages/index.js

```diff
@@ -3,24 +3,23 @@ import NumberInput from 'components/number-field/number-input';
 import Gridicon from 'components/gridicon';
 import { sanitize } from 'dompurify';
 
-const OrderPackages = ( { packages, updateValue, dimensionUnit, weightUnit, errors } ) => {
-	const renderItemInfo = ( item, itemIndex ) => (
-		<li key={ itemIndex }>
-			<span dangerouslySetInnerHTML={ { __html: sanitize( item.name ) } } />
-			{ 1 < item.quantity ? ' (x' + item.quantity + ')' : null }
-		</li>
-	);
+const renderItemInfo = ( item, itemIndex ) => (
+	<li key={ itemIndex }>
+		<span dangerouslySetInnerHTML={ { __html: sanitize( item.name ) } } />
```

Comment:
Looks like this was missed in a previous review, but same comment here as https://github.com/Automattic/woocommerce-connect-client/pull/479/files#r73906121